tests/src/main/java/io/apicurio/tests/RegistryFacade.java

```diff
@@ -19,38 +19,26 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import java.io.IOException;
-import java.net.InetSocketAddress;
-import java.net.Socket;
 import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.CompletionException;
-import java.util.concurrent.Future;
 
 public class RegistryFacade {
     static final Logger LOGGER = LoggerFactory.getLogger(RegistryFacade.class);
 
     public static final String DEFAULT_REGISTRY_JAR_PATH = "../app/target/apicurio-registry-app-1.1.2-SNAPSHOT-runner.jar";
```

Comment:
We need to do something about this - it breaks every time the version # changes.  I wonder if the default code can be updated to actually look for a pattern:  `../app/target/apicurio-registry-app-$VERSION-runner.jar`

Not something that is relevant to this PR, just an observation. :)  I'll open a ticket.

Reply:
Ticket created.

Reply:
Yeah, was thinking the same ... @Frawless any idea?

Reply:
Not sure if it's possible to get it from pom somehow. Maybe it could be parsed (it's xml) and get it from there. We do something similar in Strimzi.

Reply:
We could just use some file system code to search a directory for a file of the right pattern.